back/app/services/side_fx_user_service.rb

```diff
@@ -130,6 +130,11 @@ def create_followers(user)
     area = Area.where(id: user.domicile).first
     Follower.find_or_create_by(followable: area, user: user) if area
   end
+
+  def should_send_confirmation_email?(user)
+    user.should_send_confirmation_email? &&
```

Comment:
Nit: Since `should_send_confirmation_email?` of the User model is only used here, my suggestion would be to merge both methods into one.

Reply:
That's a great idea! Just did it https://github.com/CitizenLabDotCo/citizenlab/pull/8284/commits/1ee9d842051a8c44c24e3a4267115c5edb7b50c6